core/message_handler.go

```diff
@@ -144,7 +144,7 @@ func (app *App) HandleMessages(messages []*p2p.Message) error {
 			"from":              msg.From.String(),
 		}).Trace("not storing rejected order received from peer")
 		switch rejectedOrderInfo.Status {
-		case ROInternalError, ROOrderAlreadyStored, zeroex.RORequestFailed:
+		case ROInternalError, ROOrderAlreadyStored, zeroex.ROEthRPCRequestFailed:
```

Comment:
I think we should also add `ROCoordinatorRequestFailed` to the list of errors that should not incur a negative score?